languages/javalang/builder-renderer/java-builder-client/src/main/kotlin/io/vrap/codegen/languages/javalang/client/builder/producers/JavaModelDraftBuilderFileProducer.kt

```diff
@@ -127,7 +148,15 @@ class JavaModelDraftBuilderFileProducer @Inject constructor(override val vrapTyp
         val vrapType = vrapTypeProvider.doSwitch(this).toJavaVType() as VrapObjectType
         val constructorArguments = this.allProperties
                 .filter { it.name != this.discriminator() }
-                .map { if(it.isPatternProperty()) {"values"} else {it.name} }
+                .map {
+                    if(it.isPatternProperty()) {
```

Comment:
May be we should extract this mapping of names to a function? I guess that it sooner or later has to deal with any java reserved word. You can find a very generic version for java keywords in our current jvm sdk: https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-internal-processors/src/main/java/io/sphere/sdk/annotations/processors/generators/BaseAbstractGenerator.java#L67